can/interfaces/cantact.py

```diff
@@ -0,0 +1,135 @@
+"""
+Interface for CANtact devices from Linklayer Labs
+"""
+
+import time
+import logging
+from unittest.mock import Mock
+
+from can import BusABC, Message
+
+logger = logging.getLogger(__name__)
+
+try:
+    import cantact
+except ImportError:
+    logger.warning(
+        "The CANtact module is not installed. Install it using `python3 -m pip install cantact`"
+    )
+
+
+class CantactBus(BusABC):
+    """CANtact interface"""
+
+    @staticmethod
+    def _detect_available_configs():
+        try:
+            interface = cantact.Interface()
+        except NameError:
+            # couldn't import cantact, so no configurations are available
+            return []
+
+        channels = []
+        for i in range(0, interface.channel_count()):
+            channels.append({"interface": "cantact", "channel": "ch:%d" % i})
+        return channels
+
+    def __init__(
+        self,
+        channel,
+        bitrate=500000,
+        poll_interval=0.01,
+        monitor=False,
+        _testing=False,
+        **kwargs
+    ):
+        """
+        :param int channel:
+            Device number
```

Comment:
A short description on where to find that number would be nice. Also: Are channel lists supported?

Reply:
I'll improve the documentation for this one.